src/js/media-chrome-button.js

```diff
@@ -87,16 +88,13 @@ class MediaChromeButton extends MediaChromeElement {
     });
   }
 
-  onClick() {}
+  onClick() { }
 
   set icon(svg) {
     this.iconContainer.innerHTML = svg;
   }
 }
 
-if (!window.customElements.get('media-chrome-button')) {
-  window.customElements.define('media-chrome-button', MediaChromeButton);
-  window.MediaChromeButton = MediaChromeButton;
-}
+defineCustomElement('media-chrome-button', MediaChromeButton);
```

Comment:
This is the bulk of the changes, converting this chunk into a util function in every component file.